### PR TITLE
sfeed: init at 0.9.20

### LINKED ISSUE #105090: Packaging request: sfeed
**Project description**

From their website:
> Sfeed is a RSS and Atom parser (and some format programs).
>
> It converts RSS or Atom feeds from XML to a TAB-separated file. There are formatting programs included to convert this TAB-separated format to various other formats. There are also some programs and scripts included to import and export OPML and to fetch, filter, merge and order feed items.

**Metadata**

* homepage URL: https://codemadness.org/sfeed-simple-feed-parser.html
* source URL: https://codemadness.org/git/sfeed/log.html and `git://git.codemadness.org/sfeed`
* license: ISC
* platforms: any unix

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Motivation for this change

Closes  #105090
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

CC @humaidq for testing

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 107173` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>sfeed</li>
  </ul>
</details>

File: pkgs/tools/misc/sfeed/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  };
+
+  installPhase = ''
+    mkdir $out
+    make install PREFIX=$out
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://codemadness.org/sfeed-simple-feed-parser.html";
+    description = "a RSS and Atom parser (and some format programs)";
```
Comment (fabaff) [inline pkgs/tools/misc/sfeed/default.nix:20]:
```suggestion
    description = "RSS and Atom parser (and some format programs)";
```

Sorry, for the nitpicking. Description should start with a [capital letter](https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes).

**Comment by iblech** [comment]
I confirm that the package works here, thank you @matthiasbeyer for your work and thank you @fabaff for the review! Can this be merged?

File: pkgs/tools/misc/sfeed/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    rev = version;
+    sha256 = "17bs31wns71fx7s06rdzqkghkgv86r9d9i3814rznyzi9484c7aq";
+  };
+
+  installPhase = ''
+    mkdir $out
+    make install PREFIX=$out
+  '';
+
+  meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/sfeed/default.nix:18]:
```suggestion
  meta = with lib; {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/sfeed/default.nix [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchgit }:
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/sfeed/default.nix:1]:
```suggestion
{ stdenv, lib, fetchgit }:
```

File: pkgs/tools/misc/sfeed/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/tools/misc/sfeed/default.nix]: This is not a nit and required by the contributing guide.

**Pull request merged**